src/core/src/any.cpp

```diff
@@ -6,6 +6,9 @@
 
 #include <limits>
 #include <string>
+#include <string_view>
```

Comment:
This include seems unused

Reply:
It's actually used below: std::array<**std::string_view**, 4>